src/python_testing/TC_RR_1_1.py

```diff
@@ -13,7 +12,12 @@
 #    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
-#
+
+
+# run arguments:
+# test-runner-runs: run1
+# test-runner-run/run1/app: app/all-clusters discriminator KVS trace-to-appjson
+# test-runner-run/run1/script: storage-path commissioning-method discriminator passcode trace-to-testjson trace-to-testperfetto
```

Comment:
do not change this file yet. For testing purposes, create other files that have fixed content. That way we do not depend on tests never changing and we can test different variations without requiring tests to be changed/updated.

Reply:
Yes, I reverted back the changes to the original TC_RR_1_1.py script. Created test script TC_RR_1_1_Test.py in the same /scripts/tests/py folder